src/main/perl/RuleBasedEditor.pm

```diff
@@ -914,7 +898,8 @@ Apply configuration rules. This method is the real workhorse of the rule-based e
 
 Arguments :
     config_rules: config rules corresponding to the file to build
-    config_options: configuration parameters used to build actual configuration
+    config_options: configuration parameters used to build actual configuration. Not that keys in the
+                    config_options hash are interpreted as escaped (harmless if they are not).
```

Comment:
this is really not harmless and really not that uncommon. unescape messes up anything that is `_` followed by 2 hex values, eg. `is_active`. i really cannot wait till this is removed from pan/quattor.